cmd/fluxctl/release_cmd.go

```diff
@@ -60,16 +60,23 @@ func (opts *serviceReleaseOpts) RunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if err := checkExactlyOne("--service=<service>, or --all", opts.service != "", opts.allServices); err != nil {
+	if err := checkExactlyOne("--service=<service>, or --all", len(opts.services) > 0, opts.allServices); err != nil {
```

Comment:
What we want to check is either `--all` or _at least one_ `--service` -- this will check that, but the error message (if you supply nothing, for example) will be "please supply exactly one of .." rather than "please supply either ...".